src/test/crimson/seastore/CMakeLists.txt

```diff
@@ -15,16 +17,20 @@ add_ceph_unittest(unittest_btree_lba_manager)
 target_link_libraries(
   unittest_btree_lba_manager
   ${CMAKE_DL_LIBS}
-  crimson-seastore)
+  crimson-seastore
+  crimson-os
+  ceph-common)
 
 add_executable(unittest_seastore_journal
   test_seastore_journal.cc)
 add_ceph_test(unittest_seastore_journal
-  unittest_seastore_cache)
+  unittest_seastore_journal)
```

Comment:
this line seems strange?

Reply:
seemed like a typo (unrelated to the cmake issue)

Reply:
could you move this change into a different commit?